Makefile

```diff
@@ -12,8 +12,8 @@ test/update:
 	go test ./internal/cmd -test.update-golden
 
 dev:
-	docker buildx build . -t infrahq/infra:dev
-	docker buildx build ui -t infrahq/ui:dev
+	docker buildx build . --load -t infrahq/infra:dev
```

Comment:
The load flag explicitly loads the built image into the local store, which is something we always want in the case of local development. We had an issue where buildx was not adding the built image to the local store, this fixes it without manual dev configuration.